pkg/acs/calacs/lib/mkspt.c

```diff
@@ -77,8 +82,11 @@ int mkNewSpt (char *in_list, char *mtype, char *output) {
         return (status);
     }
 
-    /* See if an output SPT file already exists */
+    // FUTURE WORK: What if there are no input files so CALACS would not make an output file anyway?
+    /* In this case it is not necessary to See if an output SPT file already exists */
     if (FileExists (out_spt)) {
+        sprintf (MsgText, "Output SPT filename for %s exists - cannot overwrite file.", out_spt);
+        trlerror (MsgText);
         return (status);
```

Comment:
I know this was already here, but does this leak?

Reply:
This seems to be OK.

Reply:
Cool